tools/ci/ci_wptrunner_infrastructure.sh

```diff
@@ -12,19 +12,31 @@ test_infrastructure() {
     else
         ARGS=$1
     fi
-    TERM=dumb ./wpt run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $ARGS $PRODUCT infrastructure/
+    if [[ $# -eq 2 && "$2" = "--py3" ]]; then
+        TERM=dumb ./wpt $2 run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $ARGS $PRODUCT infrastructure/
+    else
+        TERM=dumb ./wpt run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $ARGS $PRODUCT infrastructure/
+    fi
 }
 
 main() {
-    PRODUCTS=( "firefox" "chrome" )
+    if [[ $# -eq 1 && "$1" = "py3" ]]; then
```

Comment:
If we just make this argument `--py3`, could we remove a bunch of the conditionals and just pass it through? This one (choosing the products) would stay, but lines 31-35 would no longer be conditional, and similarly for line 15-19 (because if an argument doesn't exist, it's just empty in bash).

So line 31 becomes:

`test_infrastructure "--binary=$(which google-chrome-unstable) --channel dev" "$1"`

And line 15:

`TERM=dumb ./wpt $2 run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $ARGS $PRODUCT infrastructure/`

Reply:
Updated the code as per review comments. Thank you!